### PR TITLE
Update repo with recent doc changes from the internal metadata

### PR DESCRIPTION
**Pull request by luisgomes252**
- [x] This contribution adheres to [CONTRIBUTING.md](https://github.com/ni/nimi-python/blob/master/CONTRIBUTING.md).
- [ ] ~~I've updated [CHANGELOG.md](https://github.com/ni/nimi-python/blob/master/CHANGELOG.md) if applicable.~~
- [ ] ~~I've added tests applicable for this pull request~~

### What does this Pull Request accomplish?

Updates nidcpower documentation to match recent changes in the internal metadata

### CODE REVIEW

**Review by marcoskirsch — APPROVED** [review]

**Comment by marcoskirsch** [comment]
ok to test

File: src/nidcpower/metadata/attributes.py [hunk 1]
```diff
@@ -1917,10 +1917,25 @@
...
+            'note': 'Refer to Supported Attributes by Device for information about supported devices.'
+        },
+        'lv_property': 'Source:Output Cutoff:Delay',
+        'name': 'OUTPUT_CUTOFF_DELAY',
+        'supported_rep_caps': [
+            'channels'
+        ],
+        'type': 'ViReal64',
+        'type_in_documentation': 'hightime.timedelta, datetime.timedelta, or float in seconds'
+    },
```
Comment (sbethur) [inline src/nidcpower/metadata/attributes.py:1934]: Unrelated change. Please revert and codegen.

**Review by sbethur — APPROVED** [review]
